lib/http_server_fixture/routes.ex

```diff
@@ -1,6 +1,8 @@
 defmodule HTTPServerFixture.Routes do
-  def routes,
-    do: %{
+  alias HTTPServer.Router.Handlers.ServeStatic
```

Comment:
I question the design of creating Handlers by aliasing other Handlers.  Perhaps the `ServeStatic` "Handler" is doing the work of _something_ other than a Handler, and should be named appropriately (and also, tested appropriatelty)

Reply:
I agree with this, naming felt hard with this one. Would it make more sense to move this out of `HTTPServer.Router.Handlers` and turn it into it's own module, like `HTTPServer.ServeStatic`? Then perhaps move `handle/1` and `get_media_type/1` to `HTTPServer.ServeStatic.Handler`?

Reply:
I think may be a good start.  The naming is always the hardest part.

Reply:
A good clue, when looking for hidden classes inside of other classes, or what to move out of one class and into the other ... start with the private functions / variables.

Reply:
Just pushed an update that creates HTTPServer.ServeStatic and changes `add_static_dir/3` to `add_static_routes/3`! HTTPServer.Router.Handlers.ServeStatic still exists in the router/handler folder with `handle/1`, but this feels like it perhaps creates greater separation between what each class is doing.